tests/pauli_rotations/rotation_test.py

```diff
@@ -271,6 +381,21 @@ def test_pauli_product_op_init(self):
         with pytest.raises(TypeError):
             PauliProductOperation(2)
 
+    # def test_remove_y_operator_not_pi_8(self):
+    #     r = PauliRotation.from_list([I, X, Y, Z, Y], Fraction(1, 4))
+    #     with pytest.raises(NotImplementedError):
+    #         r.get_y_free_equivalent()
+
+    def test_remove_y_operators_no_y_op(self):
+        r = PauliRotation.from_list([X, Z, I, Z], Fraction(1, 8))
+        assert r.get_y_free_equivalent() == ([], [])
+        assert r.ops_list == [X, Z, I, Z]
+
+    @pytest.mark.parametrize("rotation, new_rotation, lhs, rhs", test_cases_remove_y_operators())
+    def test_remove_y_operators_1(self, rotation, new_rotation, lhs, rhs):
+        assert rotation.get_y_free_equivalent() == (lhs, rhs)
+        assert rotation == new_rotation
+
```

Comment:
@alexnguyenn are these tests still required? 1. They are not passing; 2. Tests have been included seperately for PauliRotation and Measurement in their respective classes

Reply:
ops just saw that the question was @alexnguyenn

Reply:
sr I am OOTL - why are those failing now?

Reply:
It is failing because how the arguments to the test are passed. That can be fixed, but my question is if do they need to be? Previously, line 396 was 
```
assert rotation.remove_y_operator() == (lhs, rhs)
```
I thought changing it to `get_y_free_equuvalent` would fix it, But it fails because of incompatible assert statements. I now augmented the tests for both  `PauliRotations` and `Measurements` for removing y operators. Since they are already checked in the tests above, can these test be removed ?(line 384-398)

Reply:
Ok from what I can see I already removed them in https://github.com/latticesurgery-com/lattice-surgery-compiler/pull/185/commits/fcf36f7be6afa58acc01cefb32b3d89cb52fa9ac above. I am confused to why are they added back in? 

Was this caused by the merge commit you made above?

Reply:
Oh it could be. I did update my local before pushing. But I shall remove it now

Reply:
I think the problem right now is I don't know what did the merge commit change here - we might even have to revert.

Reply:
we can revert to last known checkpoint and add the changes I made in a new PR. Does that sound okay? we can revert to a80dc0f and I can add the changes I made from e407c8a to that